junit.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="1" skipped="0" tests="4" time="25.857" timestamp="2021-11-22T17:42:32.382209" hostname="MBP-van-Moritz"><testcase classname="tests.e2e.test_proof" name="test_send_proof_request" time="25.262" /><testcase classname="tests.e2e.test_proof" name="test_create_proof_request" time="0.041" /><testcase classname="tests.e2e.test_proof" name="test_accept_proof_request" time="0.060"><failure message="TypeError: must be real number, not ClientResponseError">alice_connection_id = '5a30ab64-7e5e-45f4-b132-ed857ed71e5d', async_client_alice_module_scope = &lt;httpx.AsyncClient object at 0x10f51c070&gt;
```

Comment:
I think I added this to `.gitignore`? Don't think it belongs in the PR

Reply:
hmm yeah I thought so, too, tbh.